src/test/jest/extract.test.js

```diff
@@ -4,45 +4,75 @@ const sizeof = require('object-sizeof');
 const FeatureExtractor = require('../../features/FeatureExtractor');
 const logger = require('../../logging');
 const { StatsFormat } = require('../../utils/stats-detection');
+const { strict: assert } = require('assert');
+const fs = require('fs');
 
+async function simulateConnection(dumpPath, resultPath, statsFormat) {
 
-describe('Feature extraction tests', () => {
-    beforeEach(() => {
-        // Nothing yet
-    });
+    const dumpMeta = {
+        endpointId: '1a404b1b',
+        dumpPath: dumpPath,
+        statsFormat: statsFormat
+    };
 
-    test('SFU dump feature extraction', async () => {
-        const dumpMeta = {
-            endpointId: '1a404b1b',
-            dumpPath: './src/test/jest/sfu',
-            statsFormat: StatsFormat.CHROME_STANDARD
-        };
+    const featExtractor = new FeatureExtractor(dumpMeta);
+    const results = await featExtractor.extract();
 
-        const featExtractor = new FeatureExtractor(dumpMeta);
+    const resultString = fs.readFileSync(resultPath);
+    const resultList = JSON.parse(resultString);
+    const resultTemplate = resultList.shift();
 
-        const results = await featExtractor.extract();
+    assert.deepStrictEqual(results, resultTemplate.features);
+}
 
-        logger.info('%o', results);
+describe('Feature extraction tests', () => {
 
-        expect(3).toBe(3);
+    test.skip('SFU dump feature extraction', async () => {
+        await simulateConnection(
+            './src/test/jest/sfu',
+            '',
+            StatsFormat.STANDARD
+        );
```

Comment:
Is this left for the future, where you would want to read and process a whole directory of test files?
If so a comment would helpful explaining why this is skipped for now.